src/Common/InputTypes/AutoCompleteCountry.tsx

```diff
@@ -1,34 +1,38 @@
 /* eslint-disable no-use-before-define */
-import { makeStyles } from '@material-ui/core/styles';
-import Autocomplete from '@material-ui/lab/Autocomplete';
-import React from 'react';
-import MaterialTextField from './MaterialTextField';
+import { makeStyles } from "@material-ui/core/styles";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import React from "react";
+import MaterialTextField from "./MaterialTextField";
 import { useTranslation } from "next-i18next";
-import themeProperties from './../../../styles/themeProperties';
-import { ThemeContext } from '../../../styles/themeContext';
+import themeProperties from "./../../../styles/themeProperties";
+import { ThemeContext } from "../../../styles/themeContext";
 
 // ISO 3166-1 alpha-2
 // ⚠️ No support for IE 11
 function countryToFlag(isoCode: string) {
-  return typeof String.fromCodePoint !== 'undefined'
+  return typeof String.fromCodePoint !== "undefined"
     ? isoCode
-      .toUpperCase()
-      .replace(/./g, (char) => String.fromCodePoint(char.charCodeAt(0) + 127397))
+        .toUpperCase()
+        .replace(/./g, (char) =>
+          String.fromCodePoint(char.charCodeAt(0) + 127397)
+        )
     : isoCode;
 }
 
 export default function CountrySelect(props: {
   label: React.ReactNode;
-  inputRef: ((instance: any) => void) | React.RefObject<any> | null | undefined;
+  allowedCountries?: string[];
+  inputRef?:
+    | ((instance: any) => void)
+    | React.RefObject<any>
+    | null
+    | undefined;
   name: string | undefined;
   defaultValue: String | undefined;
-  onChange:
-  | ((
-    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
-  ) => void);
+  onChange: (event: string) => void;
```

Comment:
@Shreyaschorge This change seems incorrect. Would `event` have a type of `string`?

Reply:
Renamed here bb868af182a6301e50e9398b10d4f5eecf82cf70